kie-soup-project-datamodel/kie-soup-project-datamodel-api/src/main/java/org/kie/soup/project/datamodel/oracle/ModuleDataModelOracle.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.kie.soup.project.datamodel.oracle;
```

Comment:
`org.kie.soup.project.datamodel.oracle` The package name is still the project. Would it make sense to change it to module?

Reply:
I suspect we will also have Project here sooner or later. So the project should be OK. Altought it might be better to change to workbenchproject at some point.